src/devices/ExplorerHat/category.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
please remove this file or add categories (if you remove, I'll get a warning when regenerating device listing which will remind me to pick categories). If you leave it empty device won't be listed anywhere and I won't get a warning